README.md

```diff
@@ -4,3 +4,22 @@ AzureIntegration
 Features that integrate ASP.NET Core with Azure.
 
 This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
+
+
+SiteExtensions
+===
+
+To install ASP.NET Core runtime site extension:
+
+1. Set `SCM_SITEEXTENSIONS_FEED_URL` application setting to `https://dotnet.myget.org/F/aspnetcore-release/`
```

Comment:
In the Azure portal select App Services -> your site -> Application settings, then add a new entry under Application settings with the name `SCM_SITEEXTENSIONS_FEED_URL` and value `https://dotnet.myget.org/F/aspnetcore-release/`